CHANGE-NOTES.html

```diff
@@ -1,3 +1,5 @@
 <p>Add support for IntelliJ 2022.2.</p>
 
 <p>Show a path to repository (from project root) instead of a repository name alone in the repository selection combobox.</p>
+
+<p>Show only VCS error notification when machete file parsing fails (no IDE internal error).</p>
```

Comment:
Aaand pls overwrite (not append to) the existing change notes (https://github.com/VirtusLab/git-machete-intellij-plugin/issues/896 FTW 😅)